### ISSUE TITLE
High Loss while training for Kanji Character

### ISSUE THREAD

**Issue by Zepharchit** [faustomorales/keras-ocr#46]
How can the loss be reduced?
Is there a way so that the loss could converge and results could improve.
![Screenshot from 2020-03-13 12-47-25](https://user-images.githubusercontent.com/51807559/76598464-f0b96300-6528-11ea-95f3-5a1a56652c54.png)

**Comment by faustomorales** [comment]
Looking at your screenshot, I can see the training loss is dropping steadily, so I don't think there is a misconfiguration issue with training here. I would try the following (in order).

- It could be that training just needs to go on a bit longer. Let training continue for at least 100 epochs and see if validation loss starts to drop. This means changing the `EarlyStopping` parameters to make them more permissive (e.g., increase `patience` to something much higher). Wouldn't it be nice for things to be that simple?
- It could be the training set is not diverse enough and so the model is quickly overfitting. Are you able to apply augmentation somehow to make the training set more diverse?
- It could be the validation set is too different from the training set. I wouldn't be able to help identify this without seeing the data.

I'm sorry for the trouble you're having and look forward to your feedback to try and make the package easier to work with. Thanks!

**Comment by faustomorales** [comment]
Closing due to lack of feedback. If the issue persists, please comment back and we can re-open. Thanks!

**Comment by ahsan44411** [comment]
@Zepharchit is your model working fine now. I am using coco_text_detection dataset which has 40k images but surprisingly the loss is very high just like yours. Please respond if you have found the solution.
